web/src/screens/library/LibraryPageScreen/blocks/LibraryPageTableBlock/column.ts

```
@@ -0,0 +1,201 @@
+
+    if (column.fid.startsWith("fixed:")) {
+      const fixedColumn = {
+        fixed: true,
+        fid: column.fid as `fixed:${MappableNodeField}`,
+        name: column.fid.replace("fixed:", "") as MappableNodeField,
+        type: "text" as const, // Fixed fields are always text for now
+        hidden: column.hidden,
+        _fixedFieldName: column.fid.replace("fixed:", "") as MappableNodeField,
+      } satisfies ColumnDefinitionFixed;
+
+      return [...prev, fixedColumn];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)